Jenkinsfile

```diff
@@ -6,6 +6,9 @@ pipeline {
     environment {
         branch =  sh(script: "echo ${scm.branches[0].name} | sed 's/[^a-zA-Z0-9]/_/g' ", returnStdout: true).trim()
         branch_id = "${branch}_${BUILD_ID}"
+        branch_dev = "develop" //will change to develop
+        branch_dev_artif = "rk_coverage_auto/lastSuccessfulBuild/artifact"
```

Comment:
what happens when we delete this rk_coverage_auto branch?

Reply:
We will change this one to develop before merging...this is for testing purposes